resources/zib/zib-NursingIntervention.xml

```diff
@@ -0,0 +1,454 @@
+      <constraint>
+        <key value="zib-NursingIntervention-01" />
+        <severity value="error" />
+        <human value="The start and/or end date of a repeating schedule should be communicated in CarePlan.encounter as well." />
+        <expression value="activity.detail.scheduledTiming.repeat.boundsPeriod.exists() implies (period = activity.detail.scheduledTiming.repeat.boundsPeriod)" />
+        <source value="http://nictiz.nl/fhir/StructureDefinition/zib-NursingIntervention" />
+      </constraint>
+      <mapping>
+        <identity value="zib-nursingintervention-v3.2-2020EN" />
+        <map value="NL-CM:14.2.1" />
+        <comment value="NursingIntervention" />
+      </mapping>
+    </element>
+    <element id="CarePlan.status">
+      <path value="CarePlan.status" />
+      <comment value="Sending systems that don't record an explicit status can use the following guidance to infer a value from the zib:&#xD;&#xA;&#xD;&#xA;* ProcedureEndTime present:&#xD;&#xA;    * and in the future: _active_&#xD;&#xA;    * and in the past: _completed_&#xD;&#xA;* ProcedureEndTime absent:&#xD;&#xA;    * ProcedureStartTime present: _active_&#xD;&#xA;    * ProcedureStartTime absent: _unknown_" />
+      <mapping>
+        <identity value="zib-nursingintervention-v3.2-2020EN" />
+        <map value="NL-CM:14.2.11" />
+        <comment value="ProcedureStartDateTime (implicit, main mapping is on CarePlan.period.start and optionally on CarePlan.activity.detail.scheduledTiming.repeat.boundsPeriod.start)" />
+      </mapping>
+      <mapping>
+        <identity value="zib-nursingintervention-v3.2-2020EN" />
+        <map value="NL-CM:14.2.12" />
+        <comment value="ProcedureEndDateTime (implicit, main mapping is on CarePlan.period.end and optionally on CarePlan.activity.detail.scheduledTiming.repeat.boundsPeriod.end)" />
+      </mapping>
+    </element>
+    <element id="CarePlan.intent">
+      <path value="CarePlan.intent" />
+      <definition value="Indicates the level of authority/intentionality associated with the care plan and where the care plan fits into the workflow chain.&#xD;&#xA;&#xD;&#xA;For zib NursingIntervention, this will normally be _plan_, but the context may specify a more suitable intent." />
+    </element>
+    <element id="CarePlan.category">
+      <path value="CarePlan.category" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="coding" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+      <min value="1" />
+    </element>
+    <element id="CarePlan.category:nursingIntervention">
+      <path value="CarePlan.category" />
+      <sliceName value="nursingIntervention" />
+      <min value="1" />
+      <max value="1" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="9632001" />
+        </coding>
+      </patternCodeableConcept>
+    </element>

```

Comment:
Was working on zib-MultidisciplinaryTeamMeeting, which also includes a CarePlan, and noted this. We haven't decided yet if we want to do this I guess. See  #43